docs/source/topics/ref_reserved-ip-subnets.adoc

```diff
@@ -0,0 +1,17 @@
+[id="reserved-ip-subnets_{context}"]
+= Reserved IP subnets
+
+The {prod} OpenShift cluster reserves IP subnets for internal use which should not collide with your host network.
+Ensure that the following IP subnets are available for use:
+
+.Reserved IP subnets
+* `10.217.0.0/22`
+* `10.217.4.0/23`
+* `192.168.126.0/24`
```

Comment:
Do we really required `192.168.126.0/24` one because this is something with internal dummy etcd IP and only `192.168.126.11` is what we use.

Reply:
If the user starts a pod that need to reach an IP in 192.168.126.0/24, it will not work.

Reply:
@guillaumerose but this is machine CIDR so why a pod need to reach to it?

Reply:
If I have on my LAN a MySQL server running at 192.168.126.11, I will not be able to connect to it from my CRC cluster. 

Maybe the wording is too strong:

Hypervisor networks need to be free but,
LAN networks `10.217.0.0/22`, `10.217.4.0/23`, `192.168.126.0/24` won't be reachable from the cluster.